modules/security/src/main/java/org/eclipse/xpanse/modules/security/config/AesUtil.java

```diff
@@ -0,0 +1,96 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * SPDX-FileCopyrightText: Huawei Inc.
+ *
+ */
+
+package org.eclipse.xpanse.modules.security.config;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.charset.StandardCharsets;
+import java.security.InvalidAlgorithmParameterException;
+import java.security.InvalidKeyException;
+import java.security.NoSuchAlgorithmException;
+import java.security.Security;
+import javax.crypto.BadPaddingException;
+import javax.crypto.Cipher;
+import javax.crypto.IllegalBlockSizeException;
+import javax.crypto.NoSuchPaddingException;
+import javax.crypto.SecretKey;
+import javax.crypto.spec.IvParameterSpec;
+import javax.crypto.spec.SecretKeySpec;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.commons.codec.binary.Base64;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+/**
+ * AES encryption tool class.
+ */
+@Slf4j
+@Component
+public class AesUtil {
+
+    static {
+        Security.setProperty("crypto.policy", "unlimited");
+    }
+
+    private static final String AES = "AES";
+    private static final String CIPHER_ALGORITHM = "AES/CBC/ISO10126Padding";
+
+    @Value("${aes.vi}")
+    private String vi;
+
+    @Value("${aes.key.fileName}")
+    private String aesKeyFileName;
+
+    /**
+     * AES encryption.
+     */
+    public String encode(String content) {
+        try {
+            byte[] byteEncode = content.getBytes(StandardCharsets.UTF_8);
+            byte[] byteAes = getCipher(Cipher.ENCRYPT_MODE).doFinal(byteEncode);
+            return Base64.encodeBase64String(byteAes);
+        } catch (Exception e) {
+            e.printStackTrace();
+            log.error("AES encode error, {}", e.getMessage());
+        }
+        return null;
+    }
+
+    /**
+     * AES decryption.
+     */
+    public String decode(String content) {
+        try {
+            byte[] byteContent = Base64.decodeBase64(content);
+            byte[] byteDecode = getCipher(Cipher.DECRYPT_MODE).doFinal(byteContent);
+            return new String(byteDecode, StandardCharsets.UTF_8);
+        } catch (IllegalBlockSizeException | BadPaddingException e) {
+            e.printStackTrace();
+            log.error("AES decode error, {}", e.getMessage());
+        }
+        return null;
+    }
+
+    private Cipher getCipher(int mode) {
+        Cipher cipher = null;
+        try (InputStream is =
+                new FileInputStream(System.getProperty("user.dir")+File.separator+aesKeyFileName)) {
```

Comment:
@WangLiNaruto please document this in the readme file. 
also, mention how to generate this file.